x-pack/plugins/security_solution/public/management/hooks/artifacts/use_bulk_delete_artifact.tsx

```diff
@@ -7,14 +7,19 @@
 import pMap from 'p-map';
 import { HttpFetchError } from '@kbn/core/public';
 import { ExceptionListItemSchema } from '@kbn/securitysolution-io-ts-list-types';
-import { useMutation, UseMutationResult, UseQueryOptions } from 'react-query';
+import { useMutation, UseMutationOptions, UseMutationResult } from 'react-query';
 import { ExceptionsListApiClient } from '../../services/exceptions_list/exceptions_list_api_client';
 
 const DEFAULT_OPTIONS = Object.freeze({});
 
 export function useBulkDeleteArtifact(
   exceptionListApiClient: ExceptionsListApiClient,
-  customOptions: UseQueryOptions<ExceptionListItemSchema[], HttpFetchError> = DEFAULT_OPTIONS,
+  customOptions: UseMutationOptions<
```

Comment:
I like the `customQueryOptions` name, can we change it here and elsewhere so the param names are consistent.

Reply:
🤔 Not sure that is the most appropriate name for these hooks that use `useMutation()`, since it not a "Query". 

I actually thought about changing the param name to just `options`, but ultimately left it as is.

Reply:
Hmm. Let's leave it as is for now. 👍